src/display/annotation_layer.js

```diff
@@ -1601,8 +1605,8 @@ class FreeTextAnnotationElement extends AnnotationElement {
   constructor(parameters) {
     const isRenderable = !!(
       parameters.data.hasPopup ||
-      parameters.data.title ||
-      parameters.data.contents
+      parameters.data.titleObj?.str ||
```

Comment:
I think the `?` is useless here (`a={b:1}; a.c === a?.c && a.b === a?.b`).

Reply:
If `titleObj === undefined` would ever happen, trying to access the `str`-property would throw and prevent the rest of the AnnotationLayer from rendering. Hence it seemed *overall safer* to use optional chaining everywhere instead.

Reply:
For me it wasn't possible to have a `titleObj === undefined`, but yes let's do that: it's safer.